src/v3/src/components/InputPassword/InputPassword.tsx

```diff
@@ -32,7 +32,6 @@ const InputPassword: UISchemaElementComponent<{
       messages = {},
       name,
     },
-    hideLabel,
```

Comment:
This was only added and used for one flow (security question verify). Initially I think it was thought that the question should display as a Header and the answer field should be displayed without a label. But comparing to current state of v2. The security question is displayed as a label attached to the to answer input field. 

Additionally, I removed it because if we want to really hide the label for a password component, we will not be able to use the ODY component, we will need to use generic MUI components because the ODY component does not allow the label to be hidden, it is baked into their component, and by not passing a label, it just creates an empty label element in the dom. 

We don't have a need right now to hide the label for any flow, but I figured if we do, we can introduce logic at that point (which doesn't use the ODY component since it isnt compatible).